### DISCUSSION TITLE
Allow query to specify the number of children using the Supabase API

### DISCUSSION THREAD

**Discussion by derekjwilliams** [supabase/supabase#23107] [Questions] [answered]
For example a User has many Posts, allow writing a query in Supabase API that returns the first 5 posts, something like:

```Typescript
        const { data, error } = await supabase
          .from('users')
          .select(`
            *,
            posts: posts (id, title, content, created_at, user_id, {limit: 5})
          `);
```

For now I'm making PostgrestAPI calls, which is fine, but it would be great if the above was supported

**Comment by GaryAustin1** [comment, marked as answer]
You can do this now.
https://supabase.com/docs/reference/javascript/limit?queryGroups=example&example=on-a-referenced-table
![image](https://github.com/supabase/supabase/assets/54564956/00606151-6a30-40e7-8238-472f24db5cc1)

**Reply by derekjwilliams** [reply]
Perfect!  Thanks so much!

**Comment by derekjwilliams** [comment]
GaryAustin1 answered my question, thanks
